packages/account-v2/src/App.tsx

```diff
@@ -1,15 +1,17 @@
 import React from 'react';
-import { APIProvider } from '@deriv/api';
+import { APIProvider, AuthProvider } from '@deriv/api-v2';
 import { AppOverlay } from './components/AppOverlay';
 import { RouteLinks } from './router/components/RouteLinks';
 import './index.scss';
 
 const App: React.FC = () => {
     return (
         <APIProvider standalone>
-            <AppOverlay title='Settings'>
-                <RouteLinks />
-            </AppOverlay>
+            <AuthProvider>
```

Comment:
What does this <AuthProvider> do?

Reply:
previously authorisation was done mainly within useAuthorize hook. That meant, that every component using useAuthorize hook had its own copy of authorisation state. The only shared things were items in localstorage - but they are outside of react rendering lifecycle so cannot really be used in predictable / reasonable manner to share the state. 

so now AuthProvider cotnains all the shared authorisation stated. Meaning, its way easier to keep all the logic in control and working properly. Previously we've had lots of race hazards, which are no longer there (at least ones resulting from authorizaion flow / use of hooks are no longer there)

Reply:
Thanks for the clarification 😎